Content.Client/UserInterface/Controls/RadialMenu.cs

```diff
@@ -229,6 +252,69 @@ public RadialMenuTextureButton()
         OnButtonUp += OnClicked;
     }
 
+    /// <inheritdoc />
+    protected override void Draw(DrawingHandleScreen handle)
+    {
+        var texture = TextureNormal;
+
+        if (texture == null)
```

Comment:
maybe draw colored segment even if no texture was passed?

Reply:
Yeah, that's fine. I think it actually looks better without the button texture

Reply:
now segment drawing is field that can be changed per-button (make zebra control for fun?) and button can be rendered with background, with only border etc.

Reply:
Just drop the button texture entirely, it clashes with the sector background

Reply:
Well now i understand what u meant. Honestly i was not realizing it was problem until it was gone :D